### PR TITLE
Bumpfee test fix

### LINKED ISSUE #17716: intermittent functional test failure in wallet_bumpfee
https://travis-ci.org/bitcoin/bitcoin/jobs/623304089#L8090

```
 test  2019-12-10T18:32:57.082000Z TestFramework (ERROR): Assertion failed 
                                   Traceback (most recent call last):
                                     File "/home/travis/build/bitcoin/bitcoin/build/bitcoin-s390x-unknown-linux-gnu/test/functional/test_framework/test_framework.py", line 112, in main
                                       self.run_test()
                                     File "/home/travis/build/bitcoin/bitcoin/build/bitcoin-s390x-unknown-linux-gnu/test/functional/wallet_bumpfee.py", line 90, in run_test
                                       test_small_output_with_feerate_succeeds(rbf_node, dest_address)
                                     File "/home/travis/build/bitcoin/bitcoin/build/bitcoin-s390x-unknown-linux-gnu/test/functional/wallet_bumpfee.py", line 224, in test_small_output_with_feerate_succeeds
                                       assert_equal(len(input_list), 1)
                                     File "/home/travis/build/bitcoin/bitcoin/build/bitcoin-s390x-unknown-linux-gnu/test/functional/test_framework/util.py", line 46, in assert_equal
                                       raise AssertionError("not(%s)" % " == ".join(str(arg) for arg in (thing1, thing2) + args))
                                   AssertionError: not(16 == 1)

### PR DESCRIPTION
**Pull request by instagibbs**
In the loop we accidentally used `origfee` which is not the value to check, and also allowed the loop to exit too early since the new fee must be strictly greater than `0.0005`.

Also converted/added a bunch of logging from comments.

Resolves https://github.com/bitcoin/bitcoin/issues/17716

### CODE REVIEW

File: test/functional/wallet_bumpfee.py [hunk 1]
```diff
@@ -428,15 +441,16 @@ def test_bumpfee_metadata(rbf_node, dest_address):
...
     rbf_node.walletlock()
     assert_raises_rpc_error(-13, "Please enter the wallet passphrase with walletpassphrase first.",
                             rbf_node.bumpfee, rbfid)
     rbf_node.walletpassphrase(WALLET_PASSPHRASE, WALLET_PASSPHRASE_TIMEOUT)
 
-def test_change_script_match(rbf_node, dest_address):
-    """Test that the same change addresses is used for the replacement transaction when possible."""
+def test_change_script_match(self, rbf_node, dest_address):
+    self.log.info('Test that the same change addresses is used for the replacement transaction when possible.')
     def get_change_address(tx):
```
Comment (MarcoFalke) [inline test/functional/wallet_bumpfee.py]:
```
test/functional/wallet_bumpfee.py:454:5: E306 expected 1 blank line before a nested definition, found 0

Comment (instagibbs) [reply test/functional/wallet_bumpfee.py]: fixed, linter passing, merge please

**Comment by MarcoFalke** [comment]
ACK f1b45031148105754c23af08c891387e71c3c2c3 🏈

<details><summary>Show signature and timestamp</summary>

Signature:

```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

ACK f1b45031148105754c23af08c891387e71c3c2c3 🏈
-----BEGIN PGP SIGNATURE-----

iQGzBAEBCgAdFiEE+rVPoUahrI9sLGYTzit1aX5ppUgFAlwqrYAACgkQzit1aX5p
pUgGvgwAkvzXwCDsArn/R/cQCKAPKFPl+GNz3gekt+dd27h8GPs/94lQJcFZXynQ
yetDewQNZD/jsX/Hcqm7L6SHTRF5tCAwalUxinhtppMkI1s+/WHZhkL4a2P7H/Ly
IE0j1cVwfMunbTkghFGe9d6x2kb8Spm0HUnNrPrB0CMExy/Ck+SymRpsSLprhZ93
FLbmbwcWWHoF5fydwlZpecz7h/Bi/UJB1v20YQZBSD3PXG3HnvnmRpeCM4PyM0Yw
gQNI75ms7HadiccyLpDVGJto2kSuj+WNSDV/wICKEMBxPflBjMvOEMG/BulC39V7
Pf2hvOPRUTeUxPi93ziKkd2SMDyb0UGErsJ56msUW5NBTci1EvGDUDXe8jHk9JND
BYzD5TnlKYgeT2A0YhGRHfPMjoUTtzONz85OwZKkWkF/GBUD+qFvLh9oChysuCP+
r0Zh46JtDCO3qaXFhCkuRpbGcRyY4YC6FJ9fkJV+oAjO1+5chwjGqcJNLJ8Rcir5
xGiNHth5
=hXEx
-----END PGP SIGNATURE-----
```

[Timestamp of file with hash `2cb7f2f63d873829e88b88ecaf04dec4dacf7f4a5dff0034c9a41405f4ffb2b5  -`](https://opentimestamps.org/info/?004f70656e54696d657374616d7073000050726f6f6600bf89e2e884e8929401082cb7f2f63d873829e88b88ecaf04dec4dacf7f4a5dff0034c9a41405f4ffb2b5f0109dfa9c8201b12e9c640347b0fd06280308fff010211a9b627b26ba12446cbbc245444ccf08f1045e67d8e0f0087fc80fd6b181bd500083dfe30d2ef90c8e232268747470733a2f2f6274632e63616c656e6461722e636174616c6c6178792e636f6dfff010297188bab139a2d1205b732078504eeb08f1045e67d8e0f008d9804da909c3e6a20083dfe30d2ef90c8e2e2d68747470733a2f2f616c6963652e6274632e63616c656e6461722e6f70656e74696d657374616d70732e6f7267fff01093ad687e0a7eb1bc3959d6b09db554ec08f1045e67d8e0f008ffe6876aed5b397f0083dfe30d2ef90c8e292868747470733a2f2f66696e6e65792e63616c656e6461722e657465726e69747977616c6c2e636f6df010fc540fd987182f45daf52efa416e9e4a08f1045e67d8e0f00816bfbaf2a4577e500083dfe30d2ef90c8e2c2b68747470733a2f2f626f622e6274632e63616c656e6461722e6f70656e74696d657374616d70732e6f7267)

</details>

**Review by jonatack — COMMENTED** [review]
ACK efe1bb8

**Comment by jonatack** [comment]
GitHub caches so aggressively, I didn't see this had been updated and then merged :man_shrugging:

**Pull request merged**